README.md

```diff
@@ -16,59 +16,76 @@ dotnet add package Serilog.AspNetCore
 
 **Next**, in your application's _Program.cs_ file, configure Serilog first.  A `try`/`catch` block will ensure any configuration issues are appropriately logged:
 
+_ASP.Net 6 combines the `Startup` and `Program.Main`._
+
+Update your `Program.cs` file like this
+
 ```csharp
 using Serilog;
 using Serilog.Events;
 
-public class Program
+var builder = WebApplication.CreateBuilder(args);
```

Comment:
The placement of try/catch is important. Here's the example from datalust/dotnet6-serilog-example, cut down a little bit and merged with the existing sample.

```csharp
using Serilog;

Log.Logger = new LoggerConfiguration()
    .WriteTo.Console()
    .CreateLogger();

try
{
    Log.Information("Starting web application");

    var builder = WebApplication.CreateBuilder(args);

    builder.Host.UseSerilog(); // <-- Add this line
    
    var app = builder.Build();

    app.MapGet("/", () => "Hello World!");

    app.Run();
}
catch (Exception ex)
{
    Log.Fatal(ex, "Application terminated unexpectedly");
}
finally
{
    Log.CloseAndFlush();
}
```

Other features, like Serilog request logging, configuration, and the bootstrap logger, are covered below (this one is ultra-minimal, to emphasize the super-important exception handling block).

The original example is pretty opinionated, I think we should keep the same flavor as much as possible.

Reply:
There is that better?

Reply:
Thank @darcythomas! I should have been more specific, I think we should keep the example to ☝️

Reply:
@nblumhardt 
We can do. 
I chose to do a `dotnet new razor` and then add the minimal needed to get Serilog working correctly. 
Since I thought that most people who would be using serilog in a new project would busing that template.
(Although I may be projecting there haha)

So rather than only showing the minimum,  it would be better to show the smallest needed in a 'normal' or 'common' scenario.

Thoughts?
(If I haven't changed your mind, let me know and I can update things)

Reply:
Thanks for taking a look!

I think there's value in the first README example being compact/minimal, but we do also have a web project under `samples/Sample` that would be a good place to flesh out a full example with all the details (it could be recreated now using `dotnet new razor`) - I think the README might already link to it. What do you think?

Reply:
A full sample (like https://github.com/datalust/dotnet6-serilog-example) would be very helpful.